tests/test_forges.py

```diff
@@ -0,0 +1,33 @@
+from bumpx.forge import BaseForge, GitHub
+
+
+class BaseForgeTest:
+    def test_execute_verbose(self, mocker):
+        forge = BaseForge(verbose=True)
+        execute = mocker.patch("bumpx.forge.execute")
+        forge.execute("cmd arg")
+        execute.assert_called_with("cmd arg", verbose=True)
+
+    def test_execute_quiet(self, mocker):
```

Comment:
Just for aesthetics: could we have something like @pytest.mark.parametrize to group thee two into one?

Reply:
Let me know if that makes sense!

Reply:
Fair enough, let's keep it that way :)